e2e/playwright/testing-segment-overlays.spec.ts

```diff
@@ -1463,4 +1463,328 @@ part001 = startSketchOn(XZ)
+      ).toBeFocused()
+      await page.getByRole('button', { name: 'arrow right Continue' }).click()
+
+      // Verify the Y constraint was added
+      await editor.expectEditor.toContain('center = [xAbs001, yAbs001]', {
+        shouldNormalise: true,
+      })
+
+      // Now constrain the radius
+      const radiusConstraintBtn = page.locator(
+        '[data-constraint-type="radius"][data-is-constrained="false"]'
+      )
+      await expect(radiusConstraintBtn).toBeVisible()
+      await radiusConstraintBtn.click()
+
+      // Complete the command
+      await expect(
+        page.getByTestId('cmd-bar-arg-value').getByRole('textbox')
+      ).toBeFocused()
+      await page.getByRole('button', { name: 'arrow right Continue' }).click()
+
+      // Verify all constraints were added
+      await editor.expectEditor.toContain(
+        'center = [xAbs001, yAbs001], radius = radius001',
+        { shouldNormalise: true }
+      )
+
+      // Now unconstrain the X coordinate
+      const constrainedXBtn = page.locator(
+        '[data-constraint-type="xAbsolute"][data-is-constrained="true"]'
+      )
+      await expect(constrainedXBtn).toBeVisible()
+      await constrainedXBtn.click()
+
+      // Verify the X constraint was removed
+      await editor.expectEditor.toContain(
+        'center = [345, yAbs001], radius = radius001',
+        { shouldNormalise: true }
+      )
+
+      // Now unconstrain the Y coordinate
+      const constrainedYBtn = page.locator(
+        '[data-constraint-type="yAbsolute"][data-is-constrained="true"]'
+      )
+      await expect(constrainedYBtn).toBeVisible()
+      await constrainedYBtn.click()
+
+      // Verify the Y constraint was removed
+      await editor.expectEditor.toContain(
+        'center = [345, 0], radius = radius001',
+        { shouldNormalise: true }
+      )
+
+      // Finally, unconstrain the radius
+      const constrainedRadiusBtn = page.locator(
+        '[data-constraint-type="radius"][data-is-constrained="true"]'
+      )
+      await expect(constrainedRadiusBtn).toBeVisible()
+      await constrainedRadiusBtn.click()
+
+      // Verify all constraints were removed
+      await editor.expectEditor.toContain(
+        'center = [345, 0], radius = 238.38',
+        { shouldNormalise: true }
+      )
+    })
+  })
+  test('arc with interiorAbsolute and endAbsolute kwargs overlay constraints', async ({
+    page,
+    editor,
+    homePage,
+    scene,
+    cmdBar,
+  }) => {
+    await page.addInitScript(async () => {
+      localStorage.setItem(
+        'persistCode',
+        `myvar = 141
+sketch001 = startSketchOn(XZ)
+profile001 = circleThreePoint(
+  sketch001,
+  p1 = [445.16, 202.16],
+  p2 = [445.16, 116.92],
+  p3 = [546.85, 103],
+)
+profile003 = startProfileAt([64.39, 35.16], sketch001)
+  |> line(end = [60.69, 23.02])
+  |> arc(interiorAbsolute = [159.26, 100.58], endAbsolute = [237.05, 84.07])
+  |> line(end = [70.31, 42.28])`
```

Comment:
I think something's not right. I checked out this branch and entered this code to try to verify. I'm seeing overlays are still unconstrained on the arc, and I see this in the console.

<img width="519" alt="Screenshot 2025-04-24 at 11 08 55 AM" src="https://github.com/user-attachments/assets/b06cd7df-6bc0-4bf3-8cda-790a5bdb3229" />

Reply:
Hmm, I don't seem to be able to replicate.

Reply:
https://github.com/user-attachments/assets/bfb73733-e6dd-40b9-bb4a-5dd6b6cb07c7

Were there any steps in particular you used to trigger this?

Reply:
This is fixed now, thanks guys.